hack/build/print-workspace-status.sh

```diff
@@ -50,9 +50,9 @@ RH_DEPLOY_PATH ${RH_DEPLOY_PATH:-deploy/certified-metadata-bundle/cockroach-oper
 RH_BUNDLE_VERSION ${RH_BUNDLE_VERSION:-""}
 RH_BUNDLE_IMAGE_TAG ${RH_BUNDLE_IMAGE_TAG:-${RH_BUNDLE_VERSION:-""}}
   
-IMAGE_REGISTRY ${DEV_REGISTRY:-us.gcr.io/chris-love-operator-playground}
+STABLE_IMAGE_REGISTRY ${DEV_REGISTRY:-us.gcr.io/chris-love-operator-playground}
```

Comment:
can we not hardcode a value in here please?  we're going to need to clean this up soon anyway, might as well get started if we're touching this file

Reply:
It is not harder coded but we have a default value. We are passing this value in with the Makefile, but we don’t  always use the Makefile. What would you like?

Reply:
If we don’t default if I probably need to add error handling. And these are the defaults that the developer use to make the scripts easier for them.

Reply:
There are other defaults that we may want to clean up as well, and defaults in the Makefile. Without default values it does add a layer of complexity btw.

https://github.com/cockroachdb/cockroach-operator/blob/c3914639b6f9dc1685c8cc99e0b478d0e491d077/hack/build/print-workspace-status.sh#L42